addons/hr_holidays/models/hr_employee.py

```diff
@@ -84,10 +85,12 @@ def _compute_allocation_count(self):
             ('holiday_status_id.active', '=', True),
             ('state', '=', 'validate'),
         ], ['number_of_days:sum', 'employee_id'], ['employee_id'])
-        rg_results = dict((d['employee_id'][0], d['number_of_days']) for d in data)
+        rg_results = dict((d['employee_id'][0], {"employee_id_count": d['employee_id_count'], "number_of_days": d['number_of_days']}) for d in data)
         for employee in self:
-            employee.allocation_count = float_round(rg_results.get(employee.id, 0.0), precision_digits=2)
+            result = rg_results.get(employee.id)
+            employee.allocation_count = float_round(rg_results.get(employee.id)['number_of_days'], precision_digits=2) if result else 0.0
             employee.allocation_display = "%g" % employee.allocation_count
+            employee.allocations_count = rg_results.get(employee.id)['employee_id_count'] if result else 0.0
```

Comment:
You can directly use the `result` variable
```suggestion
            employee.allocation_count = float_round(result['number_of_days'], precision_digits=2) if result else 0.0
            employee.allocation_display = "%g" % employee.allocation_count
            employee.allocations_count = result['employee_id_count'] if result else 0.0
```